pkg/zio/zeekio/parser.go

```diff
@@ -212,37 +213,40 @@ func Unflatten(columns []zeek.Column, addPath bool) ([]zeek.Column, error) {
 		cols = append(cols, newcol)
 	}
 
+	var needpath bool
 	if addPath && !hasPath {
 		pathcol := zeek.Column{Name: "_path", Type: zeek.TypeString}
 		cols = append([]zeek.Column{pathcol}, cols...)
+		needpath = true
 	}
-	return cols, nil
+	return cols, needpath, nil
```

Comment:
Nit: it looks like this never returns a non-nil error.  Since you're changing the signature here (and any dependent packages that call this function will need to be updated), lets remove the error from the signature at the same time.

Reply:
done